pom.xml

```diff
@@ -50,6 +50,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+			<groupId>org.springframework.boot</groupId>
```

Comment:
i don't think  we need this dependency?

Reply:
I also can't leave a comment but add the .ds_store to the .gitignore and remove from the git commit (google it)

Reply:
kk sounds good for some reason it didnt work on mine without that dependency but we can ignore that ill add in mine if mine doesnt work im looking into the git commit git ignore stuff and will prob do it tomorrow just wanted to reply to comments